docs/advanced.rst

```diff
@@ -46,7 +63,6 @@ Another example is related to the `feature request #143
 at the beginning and the end of a file.  This use case is easy to implement
 through a custom wrapper for the PEP 8 library::
 
-  #!python
```

Comment:
For the time being I agree with @sigmavirus24 that this should remain. I made a new branch https://github.com/PyCQA/pycodestyle/compare/gh-500 that takes your commit and re-adds this line and updates the heading format for consistency.

I'm fine to go ahead and merge that if there are no other comments?